docs/concepts/fundamentals/render_api.md

```diff
@@ -138,3 +151,23 @@ rendered = Table.render(
     request=HttpRequest(),
 )
 ```
+
+## Provide / Inject
```

Comment:
Forgot to document provide / inject when I previously added this page on "Render API"